services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/attribute/LocalTimeAttributeConverter.java

```diff
@@ -49,8 +49,9 @@
  * <p>
  * Examples:
  * <ul>
- *     <li>{@code LocalTime.of(5, 30, 0)} is stored as {@code ItemAttributeValueMapper.fromNumber("053000")}</li>
- *     <li>{@code LocalDateTime.of(5, 30, 0, 1)} is stored as {@code ItemAttributeValueMapper.fromNumber("053000.000000001")}</li>
+ *     <li>{@code LocalTime.of(5, 30, 0)} is stored as {@code ItemAttributeValueMapper.fromString("05:30:00")}</li>
+ *     <li>{@code LocalDateTime.of(5, 30, 0, 1)} is stored as </li>
+ *     <li>{@code ItemAttributeValueMapper.fromString("05:30:00.000000001")}</li>
```

Comment:
What's this?

Reply:
I was trying to let that line not exceed the length restriction. Fixed now